ntbs-service/Models/Projections/NotificationForDrugResistanceImport.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using ntbs_service.Models.Entities;
+using ntbs_service.Models.Enums;
+
+namespace ntbs_service.Models.Projections
+{
+    public interface INotificationForDrugResistanceImport
+    {
+        int NotificationId { get; }
+        DrugResistanceProfile DrugResistanceProfile { get; }
+        bool IsMdr { get; }
+        bool IsMBovis { get; }
+    }
+
+    public class NotificationForDrugResistanceImport : INotificationForDrugResistanceImport
+    {
+        public int NotificationId { get; set; }
+        public DrugResistanceProfile DrugResistanceProfile { get; set; }
+        public TreatmentRegimen? TreatmentRegimen { get; set; }
+        public Status? ExposureToKnownCaseStatus { get; set; }
+
+        // TODO:NTBS-2034 Remove duplication with NotificationDisplay
```

Comment:
I will fix this TODO if we decide to keep the change following the performance test.